src/Nethermind/Nethermind.Merge.Plugin/Handlers/V1/NewPayloadV1Handler.cs

```diff
@@ -179,7 +179,14 @@ public async Task<ResultWrapper<PayloadStatusV1>> HandleAsync(ExecutionPayloadV1
                 return NewPayloadV1Result.Syncing;
             }
 
-            if ((block.TotalDifficulty ?? 0) != 0 && (_poSSwitcher.MisconfiguredTerminalTotalDifficulty() || _poSSwitcher.BlockBeforeTerminalTotalDifficulty(parentHeader)))
+            if (_poSSwitcher.MisconfiguredTerminalTotalDifficulty())
```

Comment:
Needed for a unit test where the block.TotalDifficulty == 0 but the TTD is misconfigured.